### DISCUSSION TITLE
Resource monitor with warehouses?

### DISCUSSION THREAD

**Discussion by JavierLopezT** [Snowflake-Labs/terraform-provider-snowflake#302] [Q&A] [answered]
Hello. I would like to terraform resource monitors of warehouses. As it seems in the docs and in the code (I don't know GO so maybe I am missing something), the resource "_snowflake_resource_monitor_" has no warehouse(s) arguments. Thus, I am assuming that we can just manage resource monitors that monitor the entire account. However, the description of the argument `end_timestamp` is:
_The date and time when the resource monitor suspends the assigned warehouses._

Which explicitly says warehouses, so I am a little confused here. 

In conclusion, is it possible to monitor warehouses with _snowflake_resource_monitor_? 
Thank you very much in advance

**Comment by JavierLopezT** [comment, marked as answer]
I found out that the resource monitor is assigned in the warehouse resource itself. Case closed

**Reply by jaikumarkunduru01** [reply]
Hello @JavierLopezT

Hope you are well! I am on the same page trying to figure out how can I create resource monitors at the warehouse level. Any help would be great and appreciated.
